package.json

```diff
@@ -39,9 +39,9 @@
   "author": "Nico Westerbeck",
   "license": "ISC",
   "bugs": {
-    "url": "https://github.com/AEGEE/oms-events/issues"
+    "url": "https://github.com/AEGEE/events/issues"
   },
-  "homepage": "https://github.com/AEGEE/oms-events#readme",
+  "homepage": "https://github.com/AEGEE/events#readme",
```

Comment:
you also want to rename the repo? this will be broken till we don't

Reply:
yes

Reply:
I'll also make a PR to oms-docker changing the submodule path and vars after this will be merged